cypress/integration/test.spec.ts

```diff
@@ -3,6 +3,10 @@
  * Can and should be expanded on in more files to test more specific functionality and ensure future bugs are caught earlier
  */
 
+import { getCurrentYear, getYearRange } from '../../src/utilities';
+
+const startYear = getCurrentYear() - getYearRange();
```

Comment:
If we modify `getYearRange` to be based on the user's onboarding data (i.e. by accessing the vuex store), would that work when called in this cypress file?

Reply:
I'm not sure - but if it doesn't and we do make that change, I could instead grab the user's entrance/grad year from the onboarding modal.